mkdocs_bootswatch/flatly/css/base.css

```diff
@@ -26,7 +26,23 @@ div.source-links {
 }
 
 div.col-md-9 img {
-  max-width: 100%;
+    max-width: 100%;
+}
+
+code {
+    padding: 1px 3px;
+    background: #ecf0f1;
+    border: solid 1px #ccc;
+    color: #7b8a8b;
```

Comment:
Yeah, I was debating what to do here. However, that's the color the theme uses for code blocks. In practice, you only see that color when *not* using highlight.js though (e.g. by setting the language to be `nohighlight`).